docs/DesignDocs/SGX_QuoteEx_Integration.md

```diff
@@ -64,29 +64,44 @@ For generation of SGX evidence in ECDSA and EPID formats, the SGX quote-ex libra
 - `sgx_get_quote_size_ex(const sgx_att_key_id_t *p_att_key_id, uint32_t* p_quote_size)`
     - Return the size of the buffer needed to hold the quote to be generated for the given attestation key ID.
 - `sgx_get_quote_ex(const sgx_report_t *p_app_report, const sgx_att_key_id_t *p_att_key_id,sgx_qe_report_info_t *p_qe_report_info, uint8_t *p_quote, uint32_t quote_size)`
-    - Generate a quote for the given attestation key ID and application SGX report, and return it in the call-supplied buffer.
+    - Generate a quote for the given attestation key ID and application SGX report, and return it in the caller-supplied buffer.
 
 As compared to the DCAP library API, the quote-ex library API allows enumeration of supported evidence formats (called attestation key IDs in the API). Otherwise the quote-ex API is similar to the DCAP API, except that every function takes an input attestation key ID in its parameter list.
 
-### Host-side Plugin Library Dynamic Load of the quote-ex or DCAP Library
+### Host-side Plugin Library Load of the SGX DCAP and quote-ex Libraries
 
-Between the DCAP and quote-ex libraries:
-- The DCAP library only supports generation of SGX quote in ECDSA-p256 format. With DCAP, the quote generation can be done either in-process, or out-of-process by working with a background service running on the same platform.
-- The quote-ex library supports generation of SGX quote in multiple formats (including ECDSA-p256 and EPID variations). With quote-ex, quote generation is always done out-of-process by working with a background service (called AESM) on the local platform.
+#### Background: the SGX DCAP and quote-ex Libraries
 
-An SGX platform can have either the quote-ex library or the DCAP library, or both of them installed. When both libraries are installed, the quote-ex library takes precedence, as it supports more evidence formats.
+- The DCAP library only supports generation of SGX quotes in ECDSA-p256 format. With DCAP, the quote generation can be done either in-process, or out-of-process by working with a background service (called AESM) running on the same platform.
```

Comment:
Clarify: Is the DCAP library only usable in Data Centers (as in the DCAP acronym), or can it be used by machines outside data centers?

Reply:
The DCAP library can also be used on machines outside of data centers.

Reply:
Can we add text clarifying that, since the term DCAP seems to imply otherwise?

Reply:
Added clarification. Thanks!